taipy/core/_orchestrator/_dispatcher/_standalone_job_dispatcher.py

```diff
@@ -33,8 +34,7 @@ def __init__(self, orchestrator: _AbstractOrchestrator, subproc_initializer: Opt
         super().__init__(orchestrator)
         max_workers = Config.job_config.max_nb_of_workers or self._DEFAULT_MAX_NB_OF_WORKERS
         self._executor: Executor = ProcessPoolExecutor(
-            max_workers=max_workers,
-            initializer=subproc_initializer,
+            max_workers=max_workers, initializer=subproc_initializer, mp_context=mp.get_context("spawn")
         )  # type: ignore
```

Comment:
```suggestion
        )
```
I believe this `# type: ignore` has no effect.